### ISSUE TITLE
Combine VTIMEZONEs when combining sources

### ISSUE THREAD

**Issue by JM-Lemmi** [JM-Lemmi/ical-relay#187]
Currently only the timezone of the first Calendar will be transferred, but the need for other timezones in the file might arise when adding additional events.

**Comment by Sarsum** [comment]
@JM-Lemmi fixed by #252 isnt it?

**Comment by JM-Lemmi** [comment]
No, there must still be bugs there, but they havent been triaged yet. It doesnt work fully, or doesnt work in conjuntion with the X-WR-TIMEZONE compatibility mode.
